### DISCUSSION TITLE
Any way to get my Ducky One 3 to work?

### DISCUSSION THREAD

**Discussion by edmuff** [hrvach/deskhop#130] [General]
Hi, i love the deskhop and really want to use it. but my Ducky 3 keaboard doesnt work at all. It lights up and the board on the deskhop flashes, but no inputs work. Any ideas on making work? 
BR Niklas

**Comment by hrvach** [comment]
Sure, can you try with another keyboard first to test the board is operating properly? Are you running the latest firmware? If yes on both, another thing you can try is the DIP switch under the keyboard, switch 2 to ON (NKRO off). Any luck?

**Comment by edmuff** [comment]
Thank you for the quick reply. I have tried 2 another keyboards , a Corsair Vengeance K90 (didnt work) and a Anne Pro 2 (it works perfectly). I used the 0.62 version on both pcbs.Regardning the Ducky 3 ONE It doesnt seem to be any diffrence with the dip switch nr 2 in off or on, nor any of the other switches. The strange part is that the pcb LED behave exactly the same when all 3 keyboards are connected.z

**Comment by hrvach** [comment]
Hmmm, that sucks. Could you dump the hid descriptor (with your keyboard plugged directly into the computer), "usbhid-dump -e descriptor" and paste the output please?

**Reply by edmuff** [reply]
Unsure if i did the right thing here, but i downloaded a program that would dump USB Descriptors, here is what i got :D

Information for device Ducky One 3 RGB (VID=0x3233 PID=0x1311):

------------------------------
Connection Information:
------------------------------
Device current bus speed: FullSpeed
Device supports USB 1.1 specification
Device supports USB 2.0 specification
Device address: 0x000F
Current configuration value: 0x01
Number of open pipes: 6


------------------------------
Device Descriptor:
------------------------------
0x12	bLength
0x01	bDescriptorType
0x0110	bcdUSB
0x00	bDeviceClass      
0x00	bDeviceSubClass   
0x00	bDeviceProtocol   
0x08	bMaxPacketSize0   (8 bytes)
0x3233	idVendor
0x1311	idProduct
0x0000	bcdDevice
0x01	iManufacturer   "Ducky"
0x02	iProduct        "Ducky One 3 RGB"
0x03	iSerialNumber   "DK-V1.07-220107"
0x01	bNumConfigurations


-------------------------
Configuration Descriptor:
-------------------------
0x09	bLength
0x02	bDescriptorType
0x008D	wTotalLength   (141 bytes)
0x05	bNumInterfaces
0x01	bConfigurationValue
0x00	iConfiguration
0xA0	bmAttributes   (Bus-powered Device, Remote-Wakeup)
0x32	bMaxPower      (100 mA)

Interface Descriptor:
------------------------------
0x09	bLength
0x04	bDescriptorType
0x00	bInterfaceNumber
0x00	bAlternateSetting
0x02	bNumEndPoints
0x03	bInterfaceClass      (Human Interface Device Class)
0x00	bInterfaceSubClass   
0x00	bInterfaceProtocol   
0x00	iInterface

HID Descriptor:
------------------------------
0x09	bLength
0x21	bDescriptorType
0x0110	bcdHID
0x00	bCountryCode
0x01	bNumDescriptors
0x22	bDescriptorType   (Report descriptor)
0x001B	bDescriptorLength

Endpoint Descriptor:
------------------------------
0x07	bLength
0x05	bDescriptorType
0x81	bEndpointAddress  (IN endpoint 1)
0x03	bmAttributes      (Transfer: Interrupt / Synch: None / Usage: Data)
0x0040	wMaxPacketSize    (1 x 64 bytes)
0x01	bInterval         (1 frames)

Endpoint Descriptor:
------------------------------
0x07	bLength
0x05	bDescriptorType
0x02	bEndpointAddress  (OUT endpoint 2)
0x03	bmAttributes      (Transfer: Interrupt / Synch: None / Usage: Data)
0x0040	wMaxPacketSize    (1 x 64 bytes)
0x01	bInterval         (1 frames)

Interface Descriptor:
------------------------------
0x09	bLength
0x04	bDescriptorType
0x01	bInterfaceNumber
0x00	bAlternateSetting
0x01	bNumEndPoints
0x03	bInterfaceClass      (Human Interface Device Class)
0x00	bInterfaceSubClass   
0x01	bInterfaceProtocol   
0x00	iInterface

HID Descriptor:
------------------------------
0x09	bLength
0x21	bDescriptorType
0x0110	bcdHID
0x00	bCountryCode
0x01	bNumDescriptors
0x22	bDescriptorType   (Report descriptor)
0x0040	bDescriptorLength

Endpoint Descriptor:
------------------------------
0x07	bLength
0x05	bDescriptorType
0x83	bEndpointAddress  (IN endpoint 3)
0x03	bmAttributes      (Transfer: Interrupt / Synch: None / Usage: Data)
0x0008	wMaxPacketSize    (1 x 8 bytes)
0x01	bInterval         (1 frames)

Interface Descriptor:
------------------------------
0x09	bLength
0x04	bDescriptorType
0x02	bInterfaceNumber
0x00	bAlternateSetting
0x01	bNumEndPoints
0x03	bInterfaceClass      (Human Interface Device Class)
0x00	bInterfaceSubClass   
0x02	bInterfaceProtocol   
0x00	iInterface

HID Descriptor:
------------------------------
0x09	bLength
0x21	bDescriptorType
0x0110	bcdHID
0x00	bCountryCode
0x01	bNumDescriptors
0x22	bDescriptorType   (Report descriptor)
0x0034	bDescriptorLength

Endpoint Descriptor:
------------------------------
0x07	bLength
0x05	bDescriptorType
0x84	bEndpointAddress  (IN endpoint 4)
0x03	bmAttributes      (Transfer: Interrupt / Synch: None / Usage: Data)
0x0004	wMaxPacketSize    (1 x 4 bytes)
0x01	bInterval         (1 frames)

Interface Descriptor:
------------------------------
0x09	bLength
0x04	bDescriptorType
0x03	bInterfaceNumber
0x00	bAlternateSetting
0x01	bNumEndPoints
0x03	bInterfaceClass      (Human Interface Device Class)
0x00	bInterfaceSubClass   
0x01	bInterfaceProtocol   
0x00	iInterface

HID Descriptor:
------------------------------
0x09	bLength
0x21	bDescriptorType
0x0110	bcdHID
0x00	bCountryCode
0x01	bNumDescriptors
0x22	bDescriptorType   (Report descriptor)
0x002D	bDescriptorLength

Endpoint Descriptor:
------------------------------
0x07	bLength
0x05	bDescriptorType
0x85	bEndpointAddress  (IN endpoint 5)
0x03	bmAttributes      (Transfer: Interrupt / Synch: None / Usage: Data)
0x0040	wMaxPacketSize    (1 x 64 bytes)
0x01	bInterval         (1 frames)

Interface Descriptor:
------------------------------
0x09	bLength
0x04	bDescriptorType
0x04	bInterfaceNumber
0x00	bAlternateSetting
0x01	bNumEndPoints
0x03	bInterfaceClass      (Human Interface Device Class)
0x00	bInterfaceSubClass   
0x00	bInterfaceProtocol   
0x00	iInterface

HID Descriptor:
------------------------------
0x09	bLength
0x21	bDescriptorType
0x0110	bcdHID
0x00	bCountryCode
0x01	bNumDescriptors
0x22	bDescriptorType   (Report descriptor)
0x0039	bDescriptorLength

Endpoint Descriptor:
------------------------------
0x07	bLength
0x05	bDescriptorType
0x86	bEndpointAddress  (IN endpoint 6)
0x03	bmAttributes      (Transfer: Interrupt / Synch: None / Usage: Data)
0x0004	wMaxPacketSize    (1 x 4 bytes)
0x01	bInterval         (1 frames)

Microsoft OS Descriptor is not available. Error code: 0x0000001F


--------------------------------
String Descriptor Table
--------------------------------
Index  LANGID  String
0x00   0x0000  0x0409 
0x01   0x0409  "Ducky"
0x02   0x0409  "Ducky One 3 RGB"
0x03   0x0409  "DK-V1.07-220107"

------------------------------

Connection path for device: 
USB xHCI-kompatibel värdstyrenhet
Root Hub
Ducky One 3 RGB (VID=0x3233 PID=0x1311) Port: 8

Running on: Windows 10 or greater (Build Version 22631)

Brought to you by TDD v2.19.0, Dec  5 2023, 12:08:38

**Comment by hrvach** [comment]
Try this perhaps: https://github.com/todbot/win-hid-dump - releases section maybe has a pre-built binary...

**Reply by edmuff** [reply]
PS C:\Users\Niklas\Desktop\winhiddump> .\winhiddump.exe
2E8A:107C: Hrvoje Cavrak - DeskHop Helper
PATH:\\?\hid#vid_2e8a&pid_107c&mi_01#7&1da440a4&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  05  01  09  02  A1  01  85  05  09  01  A1  00  05  09  19  01
  29  05  15  00  25  01  35  00  45  01  65  00  55  00  75  01
  95  05  81  02  95  03  81  03  05  01  09  30  25  7F  45  00
  75  08  95  01  81  06  09  31  81  06  09  38  81  06  05  0C
  0A  38  02  81  06  C1  00  C1  00
  (73 bytes)
1038:1290: SteelSeries - Arctis Pro Wireless
PATH:\\?\hid#vid_1038&pid_1290&mi_00#9&3595ccd1&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  06  00  FF  09  01  A1  01  09  01  15  00  25  FF  35  00  45
  00  65  00  55  00  75  08  95  01  81  02  09  01  95  1F  81
  02  09  01  95  20  91  02  09  01  96  00  04  B1  02  C1  00

  (48 bytes)
2E8A:107C: Hrvoje Cavrak - DeskHop Switch
PATH:\\?\hid#vid_2e8a&pid_107c&mi_00&col01#7&3567cce9&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}\kbd
DESCRIPTOR:
  05  01  09  06  A1  01  85  01  05  07  19  E0  29  E7  15  00
  25  01  35  00  45  01  65  00  55  00  75  01  95  08  81  02
  95  38  81  03  05  08  19  01  29  05  95  05  91  02  95  03
  91  03  C1  00
  (52 bytes)
1B1C:0C0B: Corsair - Lighting Node PRO
PATH:\\?\hid#vid_1b1c&pid_0c0b#7&a62a7b3&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  06  C0  FF  09  01  A1  01  09  01  15  00  25  FF  35  00  45
  00  65  00  55  00  75  08  95  10  81  02  09  01  95  40  91
  02  C1  00
  (35 bytes)
2E8A:107C: Hrvoje Cavrak - DeskHop Switch
PATH:\\?\hid#vid_2e8a&pid_107c&mi_00&col03#7&3567cce9&0&0002#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  05  0C  09  01  A1  01  85  03  15  00  25  01  35  00  45  01
  65  00  55  00  75  01  95  20  81  03  C1  00
  (28 bytes)
1038:1290: SteelSeries - Arctis Pro Wireless
PATH:\\?\hid#vid_1038&pid_1290&mi_01#9&2f5d615f&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  06  C0  FF  09  01  A1  01  09  01  15  00  25  FF  35  00  45
  00  65  00  55  00  75  08  95  01  81  02  09  01  95  1F  81
  02  C1  00
  (35 bytes)
0B05:18AE: AsusTek Computer Inc. - ROG_RYUJIN
PATH:\\?\hid#vid_0b05&pid_18ae#7&14e9b7d0&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  06  72  FF  09  A1  A1  01  85  EC  09  10  15  00  25  FF  35
  00  45  00  65  00  55  00  75  08  95  40  81  02  09  11  91
  02  C1  00
  (35 bytes)
0B05:1A52: Generic - USB Audio
PATH:\\?\hid#vid_0b05&pid_1a52&mi_07#7&1d45eb7f&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  06  C0  FF  09  01  A1  01  85  06  06  C1  FF  09  F0  15  00
  25  FF  35  00  45  00  65  00  55  00  75  08  95  3F  81  02
  09  F1  95  01  91  02  C1  00
  (40 bytes)
3233:1311: Ducky - Ducky One 3 RGB
PATH:\\?\hid#vid_3233&pid_1311&mi_02#8&216d7652&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  05  01  09  02  A1  01  09  01  A1  00  05  09  19  01  29  05
  15  00  25  01  35  00  45  01  65  00  55  00  75  01  95  05
  81  02  95  03  81  03  05  01  09  30  25  7F  45  00  75  08
  95  01  81  06  09  31  81  06  09  38  81  06  C1  00  C1  00

  (64 bytes)
1038:1294: SteelSeries  - Arctis Pro Wireless
PATH:\\?\hid#vid_1038&pid_1294&mi_05&col02#9&26a0c965&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  06  00  FF  09  01  A1  01  85  FF  09  0A  A1  02  09  0B  17
  01  00  00  80  27  FF  FF  FF  7F  35  00  45  00  65  00  55
  00  75  20  95  01  81  02  15  00  25  01  45  01  75  01  96
  D8  01  81  03  85  90  C1  00  09  22  A1  02  09  33  25  FF
  45  00  75  08  95  01  81  02  09  34  81  02  05  01  09  3B
  26  FF  FF  75  10  81  02  25  01  45  01  75  01  96  D8  01
  81  03  85  C4  C1  00  06  00  FF  09  22  A1  02  09  33  25
  FF  45  00  75  08  95  01  81  02  09  34  81  02  05  01  09
  3B  26  FF  FF  75  10  81  02  25  01  45  01  75  01  96  D8
  01  81  03  85  E2  C1  00  06  00  FF  09  0C  A1  02  05  01
  09  3B  25  FF  45  00  75  08  95  01  81  02  25  01  45  01
  75  01  96  F0  01  81  03  85  90  C1  00  06  00  FF  09  22
  A1  02  09  33  25  FF  45  00  75  08  95  01  91  02  09  34
  91  02  05  01  09  3B  26  FF  FF  75  10  91  02  25  01  45
  01  75  01  96  D8  01  91  03  85  C4  C1  00  06  00  FF  09
  22  A1  02  09  33  25  FF  45  00  75  08  95  01  91  02  09
  34  91  02  05  01  09  3B  26  FF  FF  75  10  91  02  25  01
  45  01  75  01  96  D8  01  91  03  85  FF  C1  00  06  00  FF
  09  0C  A1  02  05  01  09  3B  25  FF  45  00  75  08  95  01
  B1  02  25  01  45  01  75  01  96  F0  01  B1  03  C1  00  C1
  00
  (321 bytes)
046D:C53A: Logitech - USB Receiver
PATH:\\?\hid#vid_046d&pid_c53a&mi_01&col01#9&3117e703&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  05  01  09  02  A1  01  85  02  09  01  A1  00  05  09  19  01
  29  10  15  00  25  01  35  00  45  01  65  00  55  00  75  01
  95  10  81  02  05  01  09  30  26  FF  7F  45  00  75  10  95
  01  81  06  09  31  81  06  09  38  25  7F  75  08  81  06  05
  0C  0A  38  02  81  06  C1  00  C1  00
  (74 bytes)
328F:0034: EMEET - HID Interface
PATH:\\?\hid#vid_328f&pid_0034&mi_04&col02#9&34b11f4&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  05  82  09  82  A1  01  85  08  09  82  25  7F  35  00  45  00
  65  00  55  00  75  08  95  7F  81  02  09  82  91  02  C1  00

  (32 bytes)
046D:C53A: Logitech - USB Receiver
PATH:\\?\hid#vid_046d&pid_c53a&mi_00#9&d40a941&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}\kbd
DESCRIPTOR:
  05  01  09  06  A1  01  05  07  19  E0  29  E7  15  00  25  01
  35  00  45  01  65  00  55  00  75  01  95  08  81  02  95  38
  81  03  05  08  19  01  29  05  95  05  91  02  95  03  91  03
  C1  00
  (50 bytes)
046D:C53A: Logitech - USB Receiver
PATH:\\?\hid#vid_046d&pid_c53a&mi_01&col02#9&3117e703&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}
Unable to parse HID Report: System.NotSupportedException: Unable to reconstruct the report descriptor.
   vid HidSharp.Platform.Windows.WinHidDevice.GetRawReportDescriptor()
   vid winhiddump.Program.Main(String[] args)DESCRIPTOR:

  (0 bytes)
2E8A:107C: Hrvoje Cavrak - DeskHop Switch
PATH:\\?\hid#vid_2e8a&pid_107c&mi_00&col02#7&3567cce9&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  05  01  09  02  A1  01  85  02  09  01  A1  00  05  09  19  01
  29  05  15  00  25  01  35  00  45  01  65  00  55  00  75  01
  95  05  81  02  95  03  81  03  05  01  09  30  26  FF  7F  45
  00  75  10  95  01  81  02  09  31  81  02  09  38  25  7F  75
  08  81  06  15  00  25  01  45  01  75  01  95  08  81  03  C1
  00  C1  00
  (83 bytes)
328F:0034: EMEET - HID Interface
PATH:\\?\hid#vid_328f&pid_0034&mi_04&col01#9&34b11f4&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  05  81  09  82  A1  01  85  07  09  82  25  7F  35  00  45  00
  65  00  55  00  75  08  95  1F  81  02  09  82  91  02  C1  00

  (32 bytes)
046D:C53A: Logitech - USB Receiver
PATH:\\?\hid#vid_046d&pid_c53a&mi_01&col03#9&3117e703&0&0002#{4d1e55b2-f16f-11cf-88cb-001111000030}
Unable to parse HID Report: System.NotSupportedException: Unable to reconstruct the report descriptor.
   vid HidSharp.Platform.Windows.WinHidDevice.GetRawReportDescriptor()
   vid winhiddump.Program.Main(String[] args)DESCRIPTOR:

  (0 bytes)
046D:C53A: Logitech - USB Receiver
PATH:\\?\hid#vid_046d&pid_c53a&mi_02&col02#9&19545abe&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  06  00  FF  09  02  A1  01  85  11  15  00  25  01  35  00  45
  01  65  00  55  00  75  01  95  98  81  03  91  03  C1  00
  (31 bytes)
2E8A:107C: Hrvoje Cavrak - DeskHop Switch
PATH:\\?\hid#vid_2e8a&pid_107c&mi_00&col04#7&3567cce9&0&0003#{4d1e55b2-f16f-11cf-88cb-001111000030}
Unable to parse HID Report: System.NotSupportedException: Unable to reconstruct the report descriptor.
   vid HidSharp.Platform.Windows.WinHidDevice.GetRawReportDescriptor()
   vid winhiddump.Program.Main(String[] args)DESCRIPTOR:

  (0 bytes)
1038:1294: SteelSeries  - Arctis Pro Wireless
PATH:\\?\hid#vid_1038&pid_1294&mi_05&col01#9&26a0c965&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  05  0C  09  01  A1  01  85  01  09  E9  15  00  25  01  35  00
  45  01  65  00  55  00  75  01  95  01  81  02  09  EA  81  02
  95  06  81  03  09  B6  95  01  81  02  09  CD  81  02  09  B5
  81  02  95  15  81  03  C1  00
  (56 bytes)
046D:C53A: Logitech - USB Receiver
PATH:\\?\hid#vid_046d&pid_c53a&mi_02&col01#9&19545abe&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  06  00  FF  09  01  A1  01  85  10  15  00  25  01  35  00  45
  01  65  00  55  00  75  01  95  30  81  03  91  03  C1  00
  (31 bytes)
0B05:19AF: AsusTek Computer Inc. - AURA LED Controller
PATH:\\?\hid#vid_0b05&pid_19af&mi_02#7&382c88b3&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  06  72  FF  09  A1  A1  01  85  EC  09  10  15  00  25  FF  35
  00  45  00  65  00  55  00  75  08  95  40  81  02  09  11  91
  02  C1  00
  (35 bytes)
3233:1311: Ducky - Ducky One 3 RGB
PATH:\\?\hid#vid_3233&pid_1311&mi_03#8&7eaedd7&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}\kbd
DESCRIPTOR:
  05  01  09  06  A1  01  05  07  19  E0  29  E7  15  00  25  01
  35  00  45  01  65  00  55  00  75  01  95  08  81  02  96  F8
  01  81  03  C1  00
  (37 bytes)
3233:1311: Ducky - Ducky One 3 RGB
PATH:\\?\hid#vid_3233&pid_1311&mi_00#8&1559c4d5&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  05  01  09  00  A1  01  15  00  25  01  35  00  45  01  65  00
  55  00  75  01  96  00  02  81  03  91  03  C1  00
  (29 bytes)
3233:1311: Ducky - Ducky One 3 RGB
PATH:\\?\hid#vid_3233&pid_1311&mi_04#8&1fae7a1c&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
DESCRIPTOR:
  05  0C  09  01  A1  01  09  E2  15  00  25  01  35  00  45  01
  65  00  55  00  75  01  95  01  81  02  09  E9  81  02  09  EA
  81  02  09  CD  81  02  09  B7  81  02  09  B5  81  02  09  B6
  81  02  0A  92  01  81  02  0A  94  01  81  02  0A  23  02  81
  02  0A  8A  01  81  02  0A  83  01  81  02  0A  21  02  81  02
  95  13  81  03  C1  00
  (86 bytes)
3233:1311: Ducky - Ducky One 3 RGB
PATH:\\?\hid#vid_3233&pid_1311&mi_01#8&39310297&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}\kbd
DESCRIPTOR:
  05  01  09  06  A1  01  05  07  19  E0  29  E7  15  00  25  01
  35  00  45  01  65  00  55  00  75  01  95  08  81  02  95  38
  81  03  05  08  19  01  29  03  95  03  91  02  95  05  91  03
  C1  00
  (50 bytes)

**Reply by edmuff** [reply]
Like this?

**Reply by hrvach** [reply]
This is officially the strangest keyboard I've seen so far. No idea what to make of it

Usage Page (Desktop),               ; Generic desktop controls (01h)
Usage (Keyboard),                   ; Keyboard (06h, application collection)
Collection (Application),
    Usage Page (Keyboard),          ; Keyboard/keypad (07h)
    Usage Minimum (KB Leftcontrol), ; Keyboard left control (E0h, dynamic value)
    Usage Maximum (KB Right GUI),   ; Keyboard right GUI (E7h, dynamic value)
    Logical Minimum (0),
    Logical Maximum (1),
    Physical Minimum (0),
    Physical Maximum (1),
    Unit,
    Unit Exponent (0),
    Report Size (1),
    Report Count (8),
    Input (Variable),
    Report Count (504),
    Input (Constant, Variable),
End Collection

NKRO without report ID - ok, but 504 bits and constant/variable? Hmmmm.

**Reply by edmuff** [reply]
Oh okey, well big thanks for checking it out. No clue what those things you posted mean :D. But it sound like its not going to work with some easy fix. If I'm going to change or buy a new keyboard, anything I should look for, to make sure it works with deskhop?

**Reply by hrvach** [reply]
I'll do my best to provide a fix in one of the future releases, because my goal is to make it as widely supported as possible ... just can't commit to any timelines because work is crazy lately. Try finding something with QMK firmware, I think it's an amazing effort for keyboards and the possibilities are endless.

**Comment by hrvach** [comment]
@edmuff would you be so kind to check/test solution suggested in https://github.com/hrvach/deskhop/issues/142#issuecomment-2308434066 and provide feedback?
